src/Alterway/Bundle/RestProblemBundle/Features/basic.feature

```diff
@@ -31,6 +31,6 @@ Feature: Basic request with errors
 
   Scenario: Catch logic exception with throw exception
     When I send a GET request to "/exception"
-    Then the response status code should be 500
+    Then the response status code should be 501
```

Comment:
https://github.com/lrqdo/RestProblemBundle/blob/0.1/test/behavior/Fixtures/Demo/src/Alterway/DemoBundle/Controller/TestController.php#L77
+
https://github.com/lrqdo/RestProblemBundle/blob/0.1/src/Alterway/Bundle/RestProblemBundle/Problem/Exception.php#L26
= 501, not 500